lib/livebook_web/live/hub_live.ex

```diff
@@ -0,0 +1,151 @@
+defmodule LivebookWeb.HubLive do
+  use LivebookWeb, :live_view
+
+  import LivebookWeb.UserHelpers
+
+  alias Livebook.Hubs
+  alias LivebookWeb.{PageHelpers, SidebarHelpers}
+  alias Phoenix.LiveView.JS
+
+  on_mount LivebookWeb.SidebarHook
+
+  @impl true
+  def mount(_params, _session, socket) do
+    {:ok,
+     assign(socket,
+       selected_provider: nil,
+       provider_id: nil,
+       hubs: [],
+       hub_options: [],
+       data: %{},
+       page_title: "Livebook - Hub"
+     )}
+  end
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div class="flex grow h-full">
+      <SidebarHelpers.sidebar
+        socket={@socket}
+        current_user={@current_user}
+        current_page=""
+        saved_hubs={@saved_hubs}
+      />
+
+      <div class="grow px-6 py-8 overflow-y-auto">
+        <div class="max-w-screen-md w-full mx-auto px-4 pb-8 space-y-8">
+          <div>
+            <PageHelpers.title text="Hub" socket={@socket} />
+            <p class="mt-4 text-gray-700">
+              Manage your Livebooks in the cloud with Hubs.
+            </p>
+          </div>
+
+          <div class="flex flex-col space-y-4">
+            <h2 class="text-xl text-gray-800 font-semibold pb-2 border-b border-gray-200">
+              1. Select your Hub service
+            </h2>
+
+            <div class="grid grid-cols-1 md:grid-cols-2 gap-4">
+              <.card_item id="fly" selected={@selected_provider} title="Fly">
+                <:logo>
+                  <%= Phoenix.HTML.raw(File.read!("static/images/fly.io.svg")) %>
```

Comment:
let's standardise on just Fly/fly?

```suggestion
                  <%= Phoenix.HTML.raw(File.read!("static/images/fly.svg")) %>
```